multigeiger/speaker.cpp

```diff
@@ -119,7 +124,8 @@ void IRAM_ATTR tick(bool high) {
   // high true: "tick" -> high frequency tick and LED blink
   // high false: "tock" -> lower frequency tock, no LED
   // called from ISR!
-  portENTER_CRITICAL_ISR(&mux_audio);
+  if (isr_sequence)
+    return;
```

Comment:
2 issues here:
- why do you check this?
- when accessing this, you need to have the mutex which you just removed from above.

Reply:
This is in a different function I didn't touch. I initally also fell for the hidden lines in Github's diff view  :)

Reply:
i checked master branch: the tick function does not have that "if" in master branch.

your branch has it, so not sure what you mean.

Reply:
obsolete